src/builtin/builtin_export.c

```diff
@@ -0,0 +1,79 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   builtin_export.c                                   :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: r3dc4t <marvin@42.fr>                      +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/11/02 00:01:18 by r3dc4t            #+#    #+#             */
+/*   Updated: 2022/11/02 00:41:52 by r3dc4t           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "builtin.h"
+#include "tab.h"
+
+static t_bool	key_is_valid(char *str)
+{
+	while (*str)
+	{
+		if (*str == SPACE_CHAR
```

Comment:
what about tabs?
maybe use
ft_isdigit && ft_isalpha
btw, this works in bash:

![image](https://user-images.githubusercontent.com/49197152/202911899-a385de2c-ef2d-49ce-bc3e-9272e34cec78.png)